src/PKSim.Infrastructure/Services/SettingsPersistor.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Serializer.Xml;
+using OSPSuite.Utility;
+using PKSim.Core;
+using PKSim.Core.Services;
+
+namespace PKSim.Infrastructure.Services
+{
+   public abstract class SettingsPersistor<TSettings> : IPersistor<TSettings>
```

Comment:
Just unified implementation./ This was exactly the same code for UserSettings and ApplicationSettings